src/device.adb

```diff
@@ -126,15 +126,15 @@ package body Device is
          return "";
       end if;
 
-      if Match ("^zfp-cortex-m(0(\+|plus|p)?|1)$") then
+      if Match ("^light-cortex-m(0(\+|plus|p)?|1)$") then
```

Comment:
Thanks @jklmnn,
Can you change the regex here to support both `zfp` and `light`?
The ZFP name is still going to be around for older compilers, for instance in the community.

Reply:
I did that in https://github.com/AdaCore/startup-gen/pull/30/commits/2c8235cfa6af3a97c8138a593a6dc3f2835b4b98.